qtsass/__init__.py

```diff
@@ -41,5 +41,21 @@
 enable_logging()
 
 # Constants
-VERSION_INFO = (0, 1, 2, 'dev0')
-__version__ = '.'.join(map(str, VERSION_INFO))
+__version__ = '0.2.2.dev0'
```

Comment:
This should be `0.2.0`, right?

Reply:
The latest tag on this repo us `0.2.2`also, it seems we never published to pypi (there is still 0.1.1) 😆

Reply:
Ok, I see.

Reply:
So good to go?